tracer/build/_build/Build.Steps.cs

```diff
@@ -879,6 +879,7 @@ void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath a
                 .Concat(includeSecurity)
                 .Select(x => Solution.GetProject(x))
                 .Where(project =>
+                project != null &&
```

Comment:
`project` should never be null here, unless we have projects that aren't in the solution? Which we shouldn't have 🤔

Reply:
🤔 I was getting a null ref when running:

```
tracer\build CompileSamples -SampleName Samples.AspNetCore5 -Framework net5.0
```

But I've just retested and it's working fine with out the `project != null` so I'll remove it.